website/addons/mendeley/static/node-cfg.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
 var CitationsNodeConfig = require('js/citationsNodeConfig').CitationsNodeConfig;
-require('./node-cfg.css');
```

Comment:
The `node-cfg.css` file in the mendeley addon can be removed as well. I do this when I merge.